### DISCUSSION TITLE
[content removed by author]

### DISCUSSION THREAD

**Discussion by hdosys** [NightscoutFoundation/xDrip#2980] [General]
[content removed by author]

**Comment by Navid200** [comment]
@tzachi-dar @keencave

**Comment by PythonFZ** [comment]
I'm also using xDrip with Juggluco and 1 min Intervall. I'd like to turn down or deactivate the smoothing xDrip does, such that it doesn't feel like xDrip is a couple of minutes behind the Juggluco values and take advantage of every minute I know earlier that I'm trending low/high.

**Comment by ah7675** [comment]
How about turn on juggluco webserver and use nightscout follow instead of Patched Libre App as hardware source?

**Reply by User-3090** [reply]
Ok, that works. How do I get the back filled data also available in xDrip? I mean, if my phone is away for some hours, I still get that data back filled in jugluco, but not in xdrip.

**Reply by User-3090** [reply]
Also, why is juggluco even required? Can't the necessary code to talk to Libre3 be integrated into xDrip+?

**Comment by hpcolus** [comment]
I cannot get xDrip to follow local webserver from juggluco. I turned juggluco webserver (Active) with and without 'local only' and in Xdrip+ as source I entered nightscout follower, url->  http://127.0.0.1:17580.  Any suggestion?

**Comment by psonnera** [comment]
Filtering Libre 2/3 data is a safety feature of xDrip+ that can be lessen to 10 minutes only (good for AAPS). If you want realtime unfiltered data use Juggluco -> Nightscout -> xDrip+ Nightscout follower.
